share/completions/tuned-adm.fish

```diff
@@ -0,0 +1,41 @@
+#disable file seletion
+complete -f tuned-adm
+
+#get profile list
+set profiles (tuned-adm list 2>&1 < /dev/null  | awk  '{print $2}' | tail -n +2 | head -n -1)
+
+# all tuned-adm actions
+set -l action list \
+        ''
```

Comment:
Please remove these lines.

In fact, please write this simply as

```fish
set -l actions list active off profile profile_info\
    recommend verify auto_profile \
    profile_mode
```

With whatever line grouping you want. Want all on one line? Sure. All on their own line? Knock yourself out. Group the lines like I did there? Okay.

Just please no "future use" spacer lines or one `set` per action.

Also note that `list` is not listed in $action**s**, because you set it to $action without the **s**.

Reply:
Sorry about that. Stupid me.

I was thinking to add descriptions. Then, I thought, it's not needed. I forgot to simplify like you did.